catherine_feijoo/class2/my_calc.rb

```diff
@@ -0,0 +1,72 @@
+def addition(num_1, num_2)
+  num_1 + num_2
+end
+
+def subtraction(num_1, num_2)
+  num_1 - num_2
+end
+
+def multiplication(num_1, num_2)
+  num_1 * num_2
+end
+
+def division(num_1, num_2)
+  num_1 / num_2
+end
+
+def modulo(num_1, num_2)
+  num_1 % num_2
+end
+
+puts "what would you like to do? ('add', 'sub', 'mult', 'div', 'mod')"
+puts" [to end enter 'end']"
+
+user_oper = ""
+
+while user_oper != "end"
+  user_oper = gets.chomp
+  
+  case
+  when user_oper == "add"
+    print "enter first number  > "
+    num_1 = gets.chomp.to_f
+    print "enter second number > "
+    num_2 = gets.chomp.to_f
```

Comment:
Since you use these 4 lines of code 5 times in this program, it may be a good idea to create a method :-)

Something like this:

```
def get_numbers
    puts "enter first number"
    num1 = gets.chomp.to_f
    puts "enter second number"
    num2 = gets.chomp.to_f
end